samples/kvsWebRTCClientMaster.c

```diff
@@ -10,6 +10,8 @@ INT32 main(INT32 argc, CHAR* argv[])
     PCHAR pChannelName;
     SignalingClientMetrics signalingClientMetrics;
     signalingClientMetrics.version = SIGNALING_CLIENT_METRICS_CURRENT_VERSION;
+    RTC_CODEC audioCodec = RTC_CODEC_OPUS;
```

Comment:
If these are the defaults, can we add a line in the readme to clarify that if the args are not provided, the default assumed by the sample is H264 and Opus?